test/helpers/cilium.go

```diff
@@ -551,7 +551,7 @@ func (s *SSHMeta) DumpCiliumCommandOutput() {
 
 	// No need to create file for bugtool because it creates an archive of files
 	// for us.
-	res := s.ExecCilium(fmt.Sprintf("bugtool -t %s", filepath.Join(BasePath, testPath)))
```

Comment:
Sorry, I didn't see this. I grep'ed for `cilium bugtool` and didn't find anything.